src/database/migrations/create-6-requests.js

```diff
@@ -0,0 +1,58 @@
+export default {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('requests', {
+    id: {
+      allowNull: false,
+      primaryKey: true,
+      type: Sequelize.UUID,
+      defaultValue: Sequelize.literal('uuid_generate_v4()')
+    },
+    departureDate: {
+      type: Sequelize.DATE,
+      allowNull: false
+    },
+    returnDate: {
+      type: Sequelize.DATE,
+      allowNull: true
+    },
+    travelReasons: {
```

Comment:
This should be in the trip table

Reply:
There would be nothing left in the request table apart from id and the other fields it joins and trip table cannot be merged with request table since we need to support multiple cities trip